### PR TITLE
chore: add node 16 on CI

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

chore

**Did you add tests for your changes?**
no

**If relevant, did you update the documentation?**
no

**Summary**
run tests on node v16

**Does this PR introduce a breaking change?**
no

**Other information**

### CODE REVIEW

File: packages/generators/init-template/webpack-template/.husky/_/husky.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  sh -e "$0" "$@"
+  exitCode="$?"
+
+  if [ $exitCode != 0 ]; then
+    echo "husky - $hook_name hook exited with code $exitCode (error)"
+    exit $exitCode
+  fi
+
+  exit 0
+fi
```
Comment (alexander-akait) [inline packages/generators/init-template/webpack-template/.husky/_/husky.sh:30]: hm , why it here?

Comment (anshumanv) [reply packages/generators/init-template/webpack-template/.husky/_/husky.sh]: old file, removed

Comment (anshumanv) [reply packages/generators/init-template/webpack-template/.husky/_/husky.sh]: old file

**Review by alexander-akait — APPROVED** [review]

**Review by evenstensberg — APPROVED** [review]

**Comment by alexander-akait** [comment]
@evenstensberg Why you merge? CI failed, it was for tests

**Comment by alexander-akait** [comment]
@anshumanv Please send again, I think we need migrate on `sass` (instead `node-sass` in some tests)

**Comment by anshumanv** [comment]
yep on it 👍

**Pull request merged**